stream_alert/rule_processor/handler.py

```diff
@@ -124,35 +138,71 @@ def run(self, event):
         # Check if debugging logging is on before json dumping alerts since
         # this can be time consuming if there are a lot of alerts
         if self._alerts and LOGGER.isEnabledFor(log_level_debug):
-            LOGGER.debug('Alerts:\n%s', json.dumps(self._alerts, indent=2))
+            LOGGER.debug('Alerts:\n%s', json.dumps(self.get_alerts(), indent=2))
 
         # Send any cached metrics to CloudWatch before returning
         self.metrics.send_metrics()
 
-        return self._failed_record_count == 0
+        return self._failed_record_count.value == 0
 
     def get_alerts(self):
         """Public method to return alerts from class. Useful for testing.
 
         Returns:
             [list] list of alerts as dictionaries
         """
-        return self._alerts
+        return self._alerts._getvalue()
```

Comment:
`pylint` will complain about accessing an internal member here; is this required?